Resolves #1420, 1429: adds start validating button to navbar

aileenzeng opened the pull request:
Resolves #1420.
Resolves #1429.

* Added a "Start Validating" button to the main navbar that takes the user to the `/validate` endpoint
* Renamed "Start Mapping" --> "Start Exploring" (from #1429)
* Added logging for "Start Validating" button, updated logging for "Start Mapping"

Before:
![image](https://user-images.githubusercontent.com/25534091/52244610-471e9180-2893-11e9-907a-efbf7081af12.png)

After:
![image](https://user-images.githubusercontent.com/25534091/52244622-53a2ea00-2893-11e9-9057-98269880d955.png)

misaugstad commented:
Shouldn't this also change the Start Mapping button that is under the "Let's create a path for everyone" text? Or did we decide to _only_ do the navbar..?

jonfroehlich commented:
Yes, I intended for the 'Start Mapping' text to change in both places--main
landing page CTA and navbar

On Mon, Feb 4, 2019 at 4:37 PM Mikey Saugstad <notifications@github.com>
wrote:

> Shouldn't this also change the Start Mapping button that is under the
> "Let's create a path for everyone" text? Or did we decide to *only* do
> the navbar..?
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/ProjectSidewalk/SidewalkWebpage/pull/1448#issuecomment-460469672>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABi-9Zi7zoyQVc_-Npg9STv927r12TnGks5vKNI2gaJpZM4aiRK5>
> .
>


-- 
Jon Froehlich
Associate Professor
Paul G. Allen School of Computer Science & Engineering
University of Washington
http://www.cs.umd.edu/~jonf/
http://makeabilitylab.io
@jonfroehlich <https://twitter.com/jonfroehlich> - Twitter

jonfroehlich commented:
Also, the 'Start Mapping' text here needs to change:

![image](https://user-images.githubusercontent.com/1621749/52288948-f1dd9100-2921-11e9-8354-2f993c879b8b.png)

misaugstad commented:
nice catch @jonfroehlich

aileenzeng commented:
Gotcha - this should all be fixed now! I changed all references of "Start Exploring --> Start Mapping" for consistency.

When modifying `help.scala.html`, I noticed that we were trying to load stuff from `public/javascripts/help/build/help.js` but it had been renamed to `public/javascripts/Help/build/help.js` at some point, so I fixed this too.

misaugstad commented:
>When modifying `help.scala.html`, I noticed that we were trying to load stuff from `public/javascripts/help/build/help.js` but it had been renamed to `public/javascripts/Help/build/help.js` at some point, so I fixed this too.

Hopefully that fixes the issue I've been having of random help build files being created whenever I switch branches that has been getting on my nerves :grin:

misaugstad commented:
awesome, looks great!
